extensions/shared/wrap-paid-block.jsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import UpgradeNudge from './upgrade-nudge';
+
+export default ( WrappedComponent, { feature, requiredPlan } ) => props => (
```

Comment:
Let's use `createHigherOrderComponent` from [`@wordpress/compose`](https://github.com/WordPress/gutenberg/tree/master/packages/compose).

Reply:
That means changing the signature (since `createHigherOrderComponent` doesn't pass through any additional arguments): a77c0a02c

What's the benefit of `createHigherOrderComponent` over just implementing the wrapping manually?

Reply:
Just `displayName` as far as I know. If you prefer to revert and wrap as you had before, please feel free.

Reply:
:man_shrugging: I guess I'm fine with the changed signature which helps highlighting which is the wrapped component and which are the props we pass to it. And a correct `displayName` doesn't hurt either.